### PR TITLE
Patch english translations

### PR DESCRIPTION
**Pull request by ehime**
Tried fixing some of the English (American) to get your point across a bit better, will need clarification on some other things (Upper vs Lower-case on some items). Hope this helps.

### CODE REVIEW

File: src/_locales/en/messages.cson [hunk 1]
```diff
@@ -37,7 +37,7 @@ openInNewWindow:
 editURL:
 	message: 'Download:'
 editURLWarning:
-	message: 'Caution: changing download location to different files will corrupt the download.'
+	message: 'Caution: Changing the downloads location will corrupt its files.'
```
Comment (jingyu9575) [inline src/_locales/en/messages.cson]: Thank you for your contribution. I'd like to clarifiy some items.
This line is shown when a task is being edited. Changing the location to a different URL of the same file (another mirror) is OK, while changing to a different file will corrupt it. This feature is to deal with a download site whose links frequently expire.
image

File: src/_locales/en/messages.cson [hunk 2]
```diff
@@ -79,31 +79,31 @@ downloadWith:
...
 	message: 'Save files to:'
 downloadFolder:
-	message: 'downloads folder'
+	message: 'Download folder'
 alwaysAsk:
-	message: 'always ask'
+	message: 'Always ask?'
 skipFirstSavingAttempt:
-	message: 'The first saving attempt is problematic and I want to skip it.'
+	message: 'The first saving attempt was problematic and was skipped'
```
Comment (jingyu9575) [inline src/_locales/en/messages.cson]: This is a checkbox option to workaround a [problem](https://github.com/jingyu9575/multithreaded-download-manager/wiki/Problems-with-disabling-the-feature-%22out-of-process-extensions%22) in some systems. When the problem happens, the first attempt to save will fail and the extension will then use an alternative slow method. Checking this option makes the extension skip the first attempt and use the alternative method directly.

Could you edit the translations to help clarifiy these items?

image

File: src/_locales/en/messages.cson (hunk 1 above)
Comment (ehime) [reply src/_locales/en/messages.cson]: cool was unaware

File: src/_locales/en/messages.cson (hunk 2 above)
Comment (ehime) [reply src/_locales/en/messages.cson]: `and should be skipped`

**Comment by jingyu9575** [comment]
Merged to v1.2a2. Thank you!

**Pull request merged**